static/js/overlays.js

```diff
@@ -63,6 +71,20 @@ exports.open_overlay = function (opts) {
 };
 
 exports.close_overlay = function (name) {
+    var overlap_overlay = _.find(overlap_overlays, function (opts) {
+        return opts.name === name;
+    });
+
+    if (overlap_overlay) {
+        overlap_overlay.overlay.removeClass("show");
+        setTimeout(function () {
+            overlap_overlay.overlay.remove();
+        }, 300);
```

Comment:
The `.overlay.show` has a 300ms animation time. This was a first-pass at it rather than an ideal solution. I'd recommend seeing actually if you can use the `ontransitionend` event with jQuery to find when the transition completes and kill the modal.